site-docker.yml.sample

```diff
@@ -1,6 +1,9 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all
+  tasks: []
+
 - hosts: mons
   become: True
```

Comment:
@leseb you bet.

On Mon, Aug 22, 2016 at 8:21 AM, Leseb <notifications@github.com> wrote:

> In site-docker.yml.sample
> <https://github.com/ceph/ceph-ansible/pull/945#discussion_r75697909>:
>
> > @@ -1,6 +1,9 @@
> >  ---
> >  # Defines deployment design and assigns role to server groups
> >
> > +- hosts: all
> > +  tasks: []
> > +
> >  - hosts: mons
> >    become: True
>
> Yup, do you mind sending a quick PR for that? I'm busy fixing the docker
> stuff :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph-ansible/pull/945/files/69cad8ecc1f5c21dd02c7c4f2590247e469c1529#r75697909>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbg94TrGstuFEaR3yfL-lssZDwq6yrOks5qib6TgaJpZM4JpoCv>
> .
>